pkg/metrics/otel.go

```diff
@@ -163,10 +163,15 @@ processors:
       # These labels are useful to users, but too noisy for global aggregation.
       - key: commit
         action: delete
-      - key: type
-        action: delete
   metricstransform/kubernetes:
     transforms:
+      - include: api_duration_seconds
+        action: update
+        operations:
+          # Eliminate 'type' tag from metric using aggregation
+          - action: aggregate_labels
+            label_set: [status, operation]
```

Comment:
This removes all labels except the ones listed, right? 
Does is also remove resource attributes?

Reply:
This lists out the remaining labels according to doc https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.54.0/processor/metricstransformprocessor#aggregate-labels.

Reply:
This is one of the downside of using metricstransform to aggregate labels, you need to call out the metric, and the labels that you want to *keep*, not *aggregated away*

Reply:
FYI the resource attributes remain, i.e. the `configsync_sync_*` tags